src/ai/backend/test/testcases/testcases.py

```diff
@@ -49,7 +49,11 @@ def __init__(self, specs: Mapping[str, TestSpec]) -> None:
 
     @classmethod
     def default(cls) -> Self:
+        # TODO: Resolve cyclic import
```

Comment:
The default function was initially a function to easily show the structure, so I think it's about time to get rid of it.
Could you separate the test case packages like you separated the auth?